egs2/TEMPLATE/asr1/asr.sh

```diff
@@ -93,7 +93,7 @@ inference_config= # Config for decoding.
 inference_args=   # Arguments for decoding, e.g., "--lm_weight 0.1".
                   # Note that it will overwrite args in inference config.
 inference_lm=valid.loss.ave.pth       # Language modle path for decoding.
-inference_asr_model=valid.acc.ave.pth # ASR model path for decoding.
+inference_asr_model=valid.acc.best.pth # ASR model path for decoding.
```

Comment:
Why did you touch this value? This changing affects all previous recipes...

Reply:
My apologies! This should not be changed. I'm sorry for that, could you please change it back?

Reply:
Okay!